<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsGcpCoveragesDialogBase</class>
 <widget class="QMainWindow" name="QgsGcpCoveragesDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>List of Gcp Coverages</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="mGcpCoveragesDatabase">
      <property name="text">
       <string>Gcp Coverages Database</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QToolBar" name="toolBar">
      <property name="windowTitle">
       <string>toolBar</string>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <attribute name="toolBarArea">
       <enum>TopToolBarArea</enum>
      </attribute>
      <attribute name="toolBarBreak">
       <bool>false</bool>
      </attribute>
      <addaction name="actionExpandAll"/>
      <addaction name="actionCollapseAll"/>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectItems</enum>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="columnCount">
       <number>3</number>
      </property>
      <attribute name="headerVisible">
       <bool>true</bool>
      </attribute>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">2</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string notr="true">3</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QDialogButtonBox" name="buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionExpandAll">
   <property name="icon">
    <iconset resource="../../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionExpandTree.svg</normaloff>:/images/themes/default/mActionExpandTree.svg</iconset>
   </property>
   <property name="text">
    <string>Expand All</string>
   </property>
   <property name="toolTip">
    <string>Show all fields of the Coverage </string>
   </property>
  </action>
  <action name="actionCollapseAll">
   <property name="icon">
    <iconset resource="../../../images/images.qrc">
     <normaloff>:/images/themes/default/mActionCollapseTree.svg</normaloff>:/images/themes/default/mActionCollapseTree.svg</iconset>
   </property>
   <property name="text">
    <string>Collapse All</string>
   </property>
   <property name="toolTip">
    <string>Show only the Coverage name</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
